src/Bids/StandardModernPrecision/BasicBids.hs

```diff
@@ -17,7 +17,7 @@ module Bids.StandardModernPrecision.BasicBids(
 import Control.Monad.Trans.State.Strict(get)
 
 import Action(Action, constrain)
-import CommonBids(cannotPreempt)
+import CommonBids(cannotPreempt, cannotPreempt2H)
```

Comment:
This name is misleading: it's for when your preemptive bids start at 2H, while 2D has some conventional meaning (Precision, Benjamin Twos, Mini-Roman, Mexican 2D, etc.). What's the right name for that? I haven't yet found one I'm happy with.